### PR TITLE
add versions for lotus and specs-actors to _meta.gen

### PR DESCRIPTION
**Pull request by nonsense**
Addresses https://github.com/filecoin-project/test-vectors/issues/7

---

TODO:
- [ ] move `.meta-gen` to another file?

### CODE REVIEW

File: gen/builders/generator.go [hunk 1]
```diff
@@ -42,9 +44,36 @@ type Generator struct {
 // genData is the generation data to stamp into vectors.
 // TODO in the future this should contain the commit of this tool and
 //  the builder api.
-var genData = GenerationData{
-	Source:  "script",
-	Version: "v0",
+var genData = []GenerationData{
+	{
+		Source:  "script",
+		Version: "v0",
```
Comment (nonsense) [inline gen/builders/generator.go:50]: @raulk I imagine we will change this on build time.

File: gen/builders/generator.go [hunk 2]
```diff
@@ -42,9 +44,36 @@ type Generator struct {
 // genData is the generation data to stamp into vectors.
 // TODO in the future this should contain the commit of this tool and
```
Comment (willscott) [inline gen/builders/generator.go:45]: remove this comment now?

File: gen/builders/generator.go [hunk 3]
```diff
@@ -42,9 +44,36 @@ type Generator struct {
...
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		panic("cant read build info")
+	}
+
+	var result []GenerationData
+
+	for _, v := range bi.Deps {
+		for _, dep := range deps {
+			if strings.HasPrefix(v.Path, dep) {
```
Comment (willscott) [inline gen/builders/generator.go:70]: are there sub-modules within lotus with their own go.mod that will also end up in here with this?

Comment (nonsense) [reply gen/builders/generator.go:70]: At the moment - no.

File: gen/builders/generator.go (hunk 1 above)
Comment (raulk) [reply gen/builders/generator.go]: Yep, we need to figure that out.

**Pull request merged**